packages/integrations/netlify/builders-types.d.ts

```diff
@@ -0,0 +1,9 @@
+declare namespace App {
+    interface Locals {
+        netlify: {
+            builders: {
+                ttl: number | undefined
+            }
+        }
+    }
+}
```

Comment:
I don't thinks this is the correct type we should export. 

When users want to type their `locals`, they do:

```ts
/// <reference types="astro/client" />
declare namespace App {
	interface Locals {
		user: {
			name: string;
			surname: string;
		};
	}
}
```

So I think we should export an interface like: 

```ts
export interface NetlifyLocals {
	netlifyBuilldersTtl: number | undefined
}
```

And the user just do :
```ts
/// <reference types="astro/client" />
declare namespace App {
	interface Locals extends NetlifyLocals {
		user: {
			name: string;
			surname: string;
		};
	}
}
```

Reply:
I like this API! I'll look into it.

Reply:
Implemented